edit/edit.go

```diff
@@ -272,17 +272,21 @@ func print(ed *Editor, a Address) ([]rune, addr, error) {
 	return rs, at, nil
 }
 
-// Where returns the rune offsets of an address.
+// Where returns the rune offsets and line offsets of an address.
 // The from offset is inclusive and to is exclusive.
 // Dot is set to the address.
-func (ed *Editor) Where(a Address) (from, to int64, err error) {
+func (ed *Editor) Where(a Address) (rfrom, rto, lfrom, lto int64, err error) {
 	ed.buf.lock.RLock()
 	defer ed.buf.lock.RUnlock()
-	at, err := where(ed, a)
-	return at.from, at.to, err
+	at, err := whereAddr(ed, a)
+	if err != nil {
+		return 0, 0, 0, 0, err
+	}
+	lfrom, lto, err = whereLine(ed, a)
+	return at.from, at.to, lfrom, lto, err
 }
 
-func where(ed *Editor, a Address) (addr, error) {
+func whereAddr(ed *Editor, a Address) (addr, error) {
```

Comment:
Doesn't matter much to me. It's not exported. Though I do slightly prefer just `where` so that it matches `Where`. I may change some of these names when I do #103.